docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,92 @@
+
+![slack connector compose](../img/connectors-slack-compose.png)
+
+## Slack API response
+
+The Slack API response will be available in a temporary local `response` variable.
+
+The following fields are available in the `response` variable. Notice that all fields describe state in the Slack workspace:
+
+- **ts**: timestamp ID
+- **channel**: channel ID
+- **message**:
+  - **type**: message type
+  - **type**: team ID
+  - **user**: user ID
+  - **text**: message text
+  - **ts**: timestamp ID
+  - **appID**: Slack App ID
+  - **botID**: Slack Bot ID
+
+You can use an Output Mapping to map the response:
+
+1. Use **Result Variable** to store the reponse in a process variable. For example `myResultVariable`
+2. Use **Result Expression** to map fields from the response into process variables. For example:
+
+```
+= {
+    messageText: response.message.text
+}
+```
+
+![slack connector response mapping](../img/connectors-slack-response-mapping.png)
+
+## Appendix
+
+To use the **Slack connector** the following prerequisites need to be set-up.
+
+1. [Slack workspace](#use-a-well-known-slack-workspace) - this is the workspace the **Slack connector** will communicate with
+2. [Slack basic app with bot token configured](#configure-a-basic-slack-app) - the **Slack connector** will communicate through this Slack app with the workspace - you can consider the Slack app as _Slack bot representing the Camunda platform_
+3. [Slack bot token stored as secret](#store-slack-bot-token-as-secret) - the secret will store the Slack bot token and can be used to reference it from BPMN processes without revealing it in the BPMN `xml`
+
+### Use a well-known Slack workspace
+
+A Slack workspace consists of channels in which workspace members can communicate and collaborate. A workspace is identified by a unique name, for example `https://myWorkspace.slack.com/`. In most case, you will already know which workspace you want to connect to. If you want to set-up a new workspace, please refer to the [official Slack documentation](https://slack.com/help/articles/115001344007-Create-a-workspace-on-Enterprise-Grid).
+
+### Configure a basic Slack app
+
+The **Slack connector** communicates through a Slack app with a concrete Slack workspace. For example, when sending a Slack message, the message will be posted by the Slack app. In order for the **Slack connector** to work, you need to perform the following steps:
+
+1. [Create a Slack app](https://api.slack.com/apps)
+2. [Request required scopes](https://api.slack.com/scopes) - the scopes represent what you app can and cannot do (for example posting messages). In order for the **Slack connector** to work, you need to at least grant the [`chat:write`](https://api.slack.com/scopes/chat:write) scope.
```

Comment:
Very minor comment, update text to:
"2. [Request required scopes](https://api.slack.com/scopes) - the scopes represent what your app can and cannot do (for example posting messages). For the **Slack connector** to work, at a minimum you need to grant the [`chat:write`](https://api.slack.com/scopes/chat:write) scope."

Reply:
yep